tunnel.go

```diff
@@ -46,8 +46,8 @@ type Tunnel struct {
 	udpConnCache *cache.Cache
 }
 
-// NewTunnel creates a Tunnel client with given options.
-func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, config *Config) (*Tunnel, error) {
+// newTunnel creates a Tunnel client with given options.
+func newTunnel(account *nkn.Account, from, to string, tuna bool, config *Config, m *nkn.MultiClient) (*Tunnel, error) {
```

Comment:
I feel like it's easier to make this functions NewTunnels()

Reply:
Yes, modified.